Rapr/DSEForm.cs

```diff
@@ -27,6 +27,8 @@ public partial class DSEForm : Form
         private Color savedForeColor;
         private OperationContext context = new OperationContext();
 
+        private static readonly IUpdateManager _updateManager = new UpdateManager();
```

Comment:
Why prefix with _?

Reply:
It's a Resharper thing, I've used it for so long I'm in the habit of using it even when Resharper is off.